docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -483,6 +545,7 @@ POST my-index/_forcemerge
 POST my-index/_shrink/my-shrunken-index
 ----
 // TEST[s/^/PUT my-index\n{"settings":{"index.number_of_shards":2,"blocks.write":true}}\n/]
+// end::free-up-disk-space[]
```

Comment:
Force-merging will probably make the disk usage worse since it has to write the new segments before it can delete the old ones. Deleting unneeded indices is the only way really.